src/components/ComparisonSidebar.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import SidebarButton from "./SidebarButton";
+
+import { ReactComponent as BackIcon} from "../assets/search-icons/back.svg";
+import { ReactComponent as CancelIcon} from "../assets/search-icons/cancel.svg"
+
+import "../css/collapsible.css";
+import "../css/sidebar.css";
+import { Link } from "react-router-dom";
+
+class ComparisonSidebar extends React.Component {
+  
+  render() {
+    const LocationList = ({locations}) => (
+      <ul className="location-list">
+        {locations.map(item => (
+          <li key={item} className="list-item">
+            <span>Location {item}</span>
+            <CancelIcon />
+          </li>
+        ))}
+      </ul>      
+    )
+
+    const locations = [1,2,3];
+
+    return (
+      <div className={`panel-container featureSelected`}>
+        <SidebarButton />
+        <div className={`sidebar-container`}>
+          <div className="sidebarHeader">
+            <div className="comparison-title">
+              <Link to="/" style={{lineHeight: 0}}>
+                <BackIcon />
+              </Link>
+              <span style={{marginLeft: 10}}>
+                Comparing Locations
+              </span>
```

Comment:
The span here is unnecessary. Its marginLeft could be a marginRight on the Link instead. Also, that margin should be 11, and lineHeight: 0 appears to do nothing.

If you set `display: flex` and `align-items: center` on the link, the back icon will vertically center itself correctly.

Also, I'd prefer we keep this CSS in sidebar.css rather than changing it up here with inline styles.

Reply:
The  lineHeight: 0 fix a issue with Link that add some px to the BackIcon height, and that make arrow unaligned.

Reply:
What about my other comments?